src/ConfigurationContext.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FunctionComponent, createContext, useContext } from "react"
+import { Platform } from "react-native"
+import env from "react-native-config"
+
+export interface Configuration {
+  displaySelfAssessment: boolean
+  displayReportAnIssue: boolean
+  appDownloadLink: string
+  authorityPrivacyPolicyUrl: string
+  authorityName: string
+  authorityAdviceUrl: string
+}
+
+const initialState = {
+  displaySelfAssessment: false,
+  displayReportAnIssue: false,
+  appDownloadLink: "",
+  authorityPrivacyPolicyUrl: "",
+  authorityName: "",
+  authorityAdviceUrl: "",
+}
+
+const ConfigurationContext = createContext<Configuration>(initialState)
+
+const DEFAULT_DOWNLOAD_LINK = ""
+const DEFAULT_AUTHORITY_PRIVACY_URL = ""
+const DEFAULT_AUTHORITY_NAME = ""
+const DEFAULT_AUTHORITY_ADVICE_URL = ""
+
+const ConfigurationProvider: FunctionComponent = ({ children }) => {
+  const authorityPrivacyPolicyUrl =
```

Comment:
Do you think this would be better or worse? Seems like it might get tough to read as this grows.

```
  const {
    PRIVACY_POLICY_URL: authorityPrivacyPolicyUrl = DEFAULT_AUTHORITY_PRIVACY_URL,
    GAEN_AUTHORITY_NAME: authorityName = DEFAULT_AUTHORITY_NAME,
    AUTHORITY_ADVICE_URL: authorityAdviceUrl = DEFAULT_AUTHORITY_ADVICE_URL,
  } = env

  const displaySelfAssessment = env.DISPLAY_SELF_ASSESSMENT === "true"
  const displayReportAnIssue = env.DISPLAY_REPORT_AN_ISSUE === "true"

  const appDownloadLink =
    Platform.select({
      ios: env.IOS_APP_STORE_URL,
      android: env.ANDROID_PLAY_STORE_URL,
    }) || DEFAULT_DOWNLOAD_LINK

```